ros_ign_image/package.xml

```diff
@@ -10,8 +10,10 @@
   <!-- Default to citadel, support blueprint -->
   <depend condition="$IGNITION_VERSION == blueprint">ignition-msgs4</depend>
   <depend condition="$IGNITION_VERSION == blueprint">ignition-transport7</depend>
-  <depend condition="$IGNITION_VERSION != blueprint">ignition-msgs5</depend>
-  <depend condition="$IGNITION_VERSION != blueprint">ignition-transport8</depend>
+  <depend condition="$IGNITION_VERSION == citadel">ignition-msgs5</depend>
+  <depend condition="$IGNITION_VERSION == citadel">ignition-transport8</depend>
+  <depend condition="$IGNITION_VERSION == dome">ignition-msgs6</depend>
+  <depend condition="$IGNITION_VERSION == dome">ignition-transport9</depend>
```

Comment:
Similar to the other comment, we need a default behavior when `IGNITION_VERSION` isn't set.

Reply:
feff3eca0a13a4fdd841291e48385a7534fb117a